### ISSUE TITLE
Error cloning repo on Windows

### ISSUE THREAD

**Issue by The11thPlague** [pineappleEA/pineapple-src#165]
When cloning the git repo on Windows, I get this error.

```
$ git clone https://github.com/pineappleEA/pineapple-src.git
Cloning into 'pineapple-src'...
remote: Enumerating objects: 120059, done.
remote: Counting objects: 100% (20829/20829), done.
remote: Compressing objects: 100% (8231/8231), done.
remote: Total 120059 (delta 14013), reused 14244 (delta 12432), pack-reused 9923
Receiving objects: 100% (120059/120059), 274.71 MiB | 1.05 MiB/s, done.

Resolving deltas: 100% (66781/66781), done.
Updating files: 100% (39563/39563), done.
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'externals/find-modules/FindLibUSB.cmake'
  'externals/find-modules/Findlibusb.cmake'
```

**Comment by SoftwareGuy** [comment]
That's not a fatal error judging from the warning. You should be OK.
Remember, some git warnings != errors.

**Comment by The11thPlague** [comment]
I'm just getting it for preservation and curiosity. The work done to develop an emulator is always interesting.
Anyway, I find it odd that there are 2 cmake modules for the same thing.

**Comment by SoftwareGuy** [comment]
Probably an error on the citrus devs' behalf. Maybe a commit failed quality checks?
See if one of the modules can be removed without making the compile error.

**Comment by MGThePro** [comment]
> I'm just getting it for preservation and curiosity. 

Probably best to use the source code from the windows release then. There was an issue with the pineapple code that meant that new code would get copied in, but deleted files wouldn't get deleted, and I never got around to fixing it. So the archive in the last windows release should be the most accurate representation of the source code (and it's what the git repo is based on anyways)

**Comment by The11thPlague** [comment]
> > I'm just getting it for preservation and curiosity.
> 
> Probably best to use the source code from the windows release then. There was an issue with the pineapple code that meant that new code would get copied in, but deleted files wouldn't get deleted, and I never got around to fixing it. So the archive in the last windows release should be the most accurate representation of the source code (and it's what the git repo is based on anyways)

Thanks. Where can I find that?
BTW Is there an equivalent for Citra?

**Comment by MGThePro** [comment]
in the releases. Right side of the page when you're looking at the main repo page.

No idea about citra

**Comment by The11thPlague** [comment]
Here? https://github.com/pineappleEA/pineapple-src/releases/tag/EA-4176
In the releases, is the code in the zip different from the code in the tar.gz?
It's not a repo though.

**Comment by MGThePro** [comment]
yes, download the Windows-Yuzu-EA-4176.zip. In there, there will be a .tar.xz with the correct source code.

**Comment by SoftwareGuy** [comment]
ZIP or Tarball doesn't matter. It is a snapshot of the repo, not the repo itself.
